requirements.txt

```diff
@@ -0,0 +1,61 @@
+Babel==2.3.4
+boto==2.45.0
+cffi==1.9.1
+changes==0.7.0
+click==3.3
+coloredlogs==5.2
+cryptography==1.7.1
+debtcollector==1.10.0
+docutils==0.13.1
+enum34==1.1.6
+funcsigs==1.0.2
+future==0.16.0
+-e git+https://github.com/witlox/gc3pie.git@4ba27c0cbf9d1d3bb85e5ddf42a039f86b802cea#egg=gc3pie
```

Comment:
Hmmm... this line fixes the GC3Pie install to a single specific commit in your repo; definitely not good for upstream.